snap/snapcraft.yaml

```diff
@@ -1,67 +1,60 @@
 name: axolotl
-summary: An signal client.
+summary: A signal client.
 description: |
   Axolotl is a crossplattform signal client
```

Comment:
If you're fixing typo, let's do it there too ;)

Reply:
You mean with the description?
I dont see an issue there, what is your proposal?

Reply:
there should be only one t to platform, shouldn't it?

Reply:
you got it. Didnt notice before